### PR TITLE
Feat/pytests

### PR DESCRIPTION
**Pull request by memorisanka**
Hi Team! Please give me your opinion, I think I've messed up terribly :P

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        run: |
+          cd Dshop
+          poetry check --no-interaction
+
+      - name: Cache dependencies
+        id: cache-deps
+        uses: actions/cache@v2.1.5
+        with:
+          path: ${{github.workspace}}/.venv
+          key: ${{ matrix.os }}-${{ hashFiles('**/poetry.lock') }}
```
Comment (jacoor) [inline .github/workflows/run_tests.yml:58]: nice!

File: .github/workflows/run_tests.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        run: |
+          cd Dshop
+          poetry config virtualenvs.in-project true
+          poetry install --no-interaction
+
+      - name: Create .env file
+        run: |
+          cd Dshop
+          echo "SECRET_KEY=default_secret_key" > .env
+          echo "DEBUG=True" >> .env
```
Comment (jacoor) [inline .github/workflows/run_tests.yml]: Please set debug=False to it mimics production. Unless it breaks.

**Review by jacoor — CHANGES_REQUESTED** [review]
Looks good!

**Review by sprzesmycki — DISMISSED** [review]
lgtm

**Review by jacoor — APPROVED** [review]

**Pull request merged**